docs/source/topics/configfile.rst

```diff
@@ -34,13 +34,16 @@ information about chalice stages.
   key will automatically be created for you.  See the examples
   section below for some stage specific configurations.
 
+* ``api_gateway_endpoint_type``  - The endpoint configuration of the
+  deployed API Gateway, can be EDGE, REGIONAL, PRIVATE. Note this
```

Comment:
Might be nice to either expand on what those things mean and what the default is. Or maybe link to the docs where it explains the difference.

Reply:
thanks for the review/feedback. I'll update to note the default and link to endpoint type docs.